explorer/client/src/pages/transaction-result/TransactionView.tsx

```diff
@@ -150,9 +158,24 @@ type TxItemView = {
 function ItemView({ data }: { data: TxItemView }) {
     return (
         <div className={styles.itemView}>
-            <div className={styles.itemviewtitle}>{data.title}</div>
+            <div
+                className={
+                    data.titleStyle
+                        ? styles[data.titleStyle]
+                        : styles.itemviewtitle
+                }
+            >
+                {data.title}
+            </div>
             <div className={styles.itemviewcontent}>
                 {data.content.map((item, index) => {
+                    // handle sender -> recipient display in one line
+                    let links: Link[] = [];
+                    if (Array.isArray(item)) {
+                        links = getAddressesLinks(item);
+                        item.label = 'Sender, Recipient';
```

Comment:
This mutates an object provided via a prop, which we shouldn't do. Maybe instead we can set a variable?

```typescript
let links: Link[] = [];
let label = item.label;
if (Array.isArray(item)) {
  links = getAddressesLinks(item);
  label = 'Sender, Recipient';
}
```